### PR TITLE
Additional explaination for the training loop

### PR DESCRIPTION
**Pull request by tberends**
I took me sometime to understand the zero-gradient step with respect to the optimizer.step-function. This step is in many videos like the 3Blue1Brown videos not mentioned because only one training step is outlined. This small adjusts graps the essence I hope. Keep up the good work!

### CODE REVIEW

**Comment by mrdbourke** [comment]
Thank you, this is a great clarification. I appreciate it!
